compiler/rustc_mir_build/src/thir/mod.rs

```diff
@@ -283,6 +283,7 @@ crate enum ExprKind<'tcx> {
         substs: UpvarSubsts<'tcx>,
         upvars: Vec<ExprRef<'tcx>>,
         movability: Option<hir::Movability>,
+        opt_fake_reads: Option<Vec<(ExprRef<'tcx>, FakeReadCause)>>,
```

Comment:
Does this have to be an Option? Can't this just be an empty vec in that case?

Reply:
Lol, you asked me to make it an Option